image/golem_blender_app/requirements.txt

```diff
@@ -5,7 +5,7 @@ numpy==1.15.4
 opencv-python==4.0.0.21
 OpenEXR==1.3.2
 peewee==3.11.2
-Pillow==5.1.0
+Pillow==6.2.0
```

Comment:
@maaktweluit have you confirmed the image works correctly with the upgraded version of Pillow?

Reply:
this change was taken from golem-core https://github.com/golemfactory/golem/pull/4875

I have not checked if the image works, i assumed the CGI team had already done this.